org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -290,6 +319,10 @@ protected void copyPO(PO copy)
 	
 	/** Immutable flag **/
 	private boolean m_isImmutable = false;
+	
+	private String m_uuid = null;
+	/** UU flag **/
+	private boolean m_isUUID = false;
```

Comment:
should avoid adding another new flag here - increase potential for side effect and complicate po state management.